examples/meta/src/regression/least_squares_regression.sg

```diff
@@ -0,0 +1,33 @@
+CSVFile f_feats_train("../../data/regression_1d_linear_features_train.dat")
+CSVFile f_feats_test("../../data/regression_1d_linear_features_test.dat")
+CSVFile f_labels_train("../../data/regression_1d_linear_labels_train.dat")
+CSVFile f_labels_test("../../data/regression_1d_linear_labels_test.dat")
+
+#![create_features]
+RealFeatures features_train(f_feats_train)
+RealFeatures features_test(f_feats_test)
+RegressionLabels labels_train(f_labels_train)
+RegressionLabels labels_test(f_labels_test)
+#![create_features]
+
+#![create_instance]
+LeastSquaresRegression lsr(features_train, labels_train)
+#![create_instance]
+
+#![train_and_apply]
+lsr.train()
+RegressionLabels labels_predict = lsr.apply_regression(features_test)
+Real bias = lsr.get_bias()
+#![train_and_apply]
+
+#[!disable_bias]
+lsr.set_compute_bias(False)
+#[!disable_bias]
+
+#![evaluate_accuracy]
+MeanSquaredError mse()
+Real accuracy = mse.evaluate(labels_predict, labels_test)
```

Comment:
this variable should not be called accuracy, but mse.
And your mse should be called evaluation